qutebrowser/browser/webelem.py

```diff
@@ -50,6 +50,7 @@
     Group.url: '[src], [href]',
     Group.prevnext: 'a, area, button, link, [role=button]',
     Group.focus: '*:focus',
+    Group.inputs: 'input[type=text], input[type=password], textarea',
```

Comment:
Why only `text` and `password` as types, instead of `input:not([type=hidden])` like above in `Group.all`?